flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.configuration;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.kubernetes.cli.KubernetesCliOptions;
+
+import static org.apache.flink.configuration.ConfigOptions.key;
+
+/**
+ * This class holds configuration constants used by Flink's kubernetes runners.
+ */
+public class KubernetesConfigOptions {
+
+	public static final ConfigOption<String> REST_SERVICE_EXPOSED_TYPE =
+		key("kubernetes.rest-service.exposed.type")
+		.defaultValue(ServiceExposedType.LoadBalancer.toString())
+		.withDescription("It could be ClusterIP/NodePort/LoadBalancer(default). When set to ClusterIP, the rest service" +
+				"will not be created.");
+
+	public static final ConfigOption<String> JOB_MANAGER_SERVICE_ACCOUNT =
+		key("kubernetes.jobmanager.service-account")
+		.defaultValue("default")
+		.withDescription("Service account that is used by jobmanager within kubernetes cluster. " +
+			"The job manager uses this service account when requesting taskmanager pods from the API server.");
+
+	public static final ConfigOption<Double> JOB_MANAGER_CPU =
+		key("kubernetes.jobmanager.cpu")
+		.defaultValue(1.0)
+		.withDescription("The number of cpu used by job manager");
+
+	public static final ConfigOption<Double> TASK_MANAGER_CPU =
+		key("kubernetes.taskmanager.cpu")
+		.defaultValue(1.0)
+		.withDescription("The number of cpu used by task manager");
+
+	public static final ConfigOption<String> CONTAINER_IMAGE_PULL_POLICY =
+		key("kubernetes.container.image.pullPolicy")
+		.defaultValue("Always")
+		.withDescription("Kubernetes image pull policy. Valid values are Always, Never, and IfNotPresent.");
+
+	public static final ConfigOption<String> KUBE_CONFIG_FILE =
+		key("kubernetes.config.file")
+		.noDefaultValue()
+		.withDescription("The kubernetes config file will be used to create the client. The default " +
+				"is located at ~/.kube/config");
+
+	public static final ConfigOption<String> NAME_SPACE =
```

Comment:
namespace is a word, maybe we don't need to seperate this word ?

Reply:
Nice catch. I will fix it.